docs/source/deployment_guide_overview.md

```diff
@@ -0,0 +1,173 @@
+
+While all of the certificates associated with an organization can be created by a single "root" CA (that is, a CA that is its own root of trust), for added security organizations can decide to use "intermediate" CAs (for example, two intermediate CAs, one for org enrollment and another for TLS) whose certificates are created by a root CA (or another intermediate CA that eventually leads back to a root CA). Because a compromise in the root CA leads to a collapse for its entire trust domain (the certs for the admins, nodes, and any CAs it has generated certificates for), intermediate CAs are a useful way to limit the exposure of the root CA. Whether you choose to use intermediate CAs will depend on the needs of your use case. They are not mandatory.
+
+There are two main types of CAs, each with a related role in generated the certificates associated with an organization.
+
+* One CA (or more, if you are using intermediate CAs) is used to generate (through a process called "enrollment") the certificates of the admin of an organization, the MSP of that organization, and any nodes owned by that organization. This CA will also generate the certificates for any additional users. Because of its role in "enrolling" identities, this CA is sometimes called the "enrollment CA" or the "ecert CA".
+* The other CA generates the certificates used to secure communications on Transport Layer Security (TLS). For this reason, this CA is often referred to as a "TLS CA". These TLS certificates are attached to actions as a way of preventing "man in the middle" attacks. Note that the TLS CA is only used for issuing certificates and can be shut down when that activity is completed. Users have the option to use one way (server only) TLS as well as two way (server and client) TLS, with the latter also known as "mutual TLS". Note that mutual TLS is necessary for [service discovery](discovery-overview.html) and [private data collections](private-data/private-data.html) to work. **Because specifying that your network will be using TLS (which is recommended) should be decided before deploying the "enrollment" CA (the YAML file specifying the configuration of this CA has a field for enabling TLS), you should deploy your TLS CA first and use it to generate a certificate that will be used when bootstrapping your enrollment CA.** Note that organizations can decide to share a TLS CA for the generation of TLS certificates if they wish. However, this increases the exposure of the TLS CA in case. If all of the organizations in a network use a single TLS CA and this CA is compromised, it might not be possible to change the configuration of the network to enable new TLS certificates to be used on the network.
```

Comment:
> Note that mutual TLS is necessary for [service discovery](discovery-overview.html) and [private data collections](private-data/private-data.html) to work.

Is this really true? Mutual TLS isn't enabled in BYFN or test-network (or many other networks) and both service discovery and private data collections seem to work just fine.